charts/kong/README.md

```diff
@@ -442,6 +446,26 @@ documentation on Service
 DNS](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/)
 for more detail.
 
+### Cert Manager Integration
+
+Kong services and proxied services have various certificate requirements.
+By default, this chart installs Kong without [cert-manager](https://cert-manager.io/docs/) integration and
+dataplanes generate a fallback self-signed certificate on each container if
+no other source for the default proxy certificate is explicitly provided.
+
+This cert-manager support requires that cert-manager be installed and
+a public or [self-signed](https://cert-manager.io/docs/configuration/selfsigned/) valid chain-of-trust issuer be ready before
+deploying Kong.
```

Comment:
If I'm not mistaken we support using specific issuers, but we'll fall back to a _default issuer_ if one is available. If that's the case is that worth pointing out here to clarify?

Reply:
in-discussion on comment https://github.com/Kong/charts/pull/592#discussion_r938189394